tests/odbc/test_odbc.py

```diff
@@ -0,0 +1,104 @@
+import pandas as pd
+import psycopg2
+import pyodbc
+import pytest
+from pydantic import ValidationError
+
+from toucan_connectors.odbc.odbc_connector import OdbcConnector, OdbcDataSource
+
+
+def test_postgres_driver_installed():
+    """
+    This test aims to check that pgodbc is installed on the server
```

Comment:
I recommend keeping docstrings as short as possible, with direct active tone:
```suggestion
    Check that pgodbc is installed
```